Jint.Repl/Program.cs

```diff
@@ -57,18 +57,22 @@ private static void Main(string[] args)
                 Console.ForegroundColor = defaultColor;
                 Console.Write("jint> ");
                 var input = Console.ReadLine();
-                if (input == "exit")
+                if (input is "exit" or ".exit")
```

Comment:
I made the command line a bit more node like, wanted to also see statement execution results like in node console